tests/plugins/test_add_filesystem.py

```diff
@@ -221,11 +215,9 @@ def make_and_store_reactor_config_map(workflow, additional_koji=None):
     workflow.conf.conf = reactor_map
 
 
+@pytest.mark.skip
```

Comment:
Could you add reasons for the skipped tests? `@pytest.mark.skip(reason="...")`

Reply:
added reason for all skipped tests